packages/serializer/src/index.ts

```diff
@@ -0,0 +1,101 @@
+import _ from "lodash"
+
+const isSerializable = (value: any) => {
+  const nestedSerialisable: any = (v: any) =>
+    (_.isPlainObject(v) || Array.isArray(v)) && _.every(v, isSerializable)
+
+  return _.overSome([
+    _.isUndefined,
+    _.isNull,
+    _.isBoolean,
+    _.isNumber,
+    _.isString,
+    nestedSerialisable,
+  ])(value)
+}
+
+export const serialize = (input: any) => {
+  let json: any = {}
+  let meta: any = {}
```

Comment:
**suggestion**: Should we type that to be at least `object`? Both the json and the meta should be at least objects. The `input` can be an optional object.

Reply:
Oh yeah, nice idea. Fixed 😄